cmd/update-notifier.go

```diff
@@ -27,6 +27,61 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
+func humanizeDuration(duration time.Duration) (durationStr string) {
```

Comment:
Already showed you how to address this without changing the UI and also remove custom code @balamurugana